spec/support/work_fixtures.rb

```diff
@@ -60,10 +60,14 @@ def contact_emails_fixture
 def keywords_fixture
   [
     {
-      'text' => 'Biology'
+      'text' => 'Biology',
+      'uri' => 'http://id.worldcat.org/fast/832383/',
+      'cocina_type' => 'topic'
     },
     {
-      'text' => 'Chemistry'
+      'text' => 'Chemistry',
```

Comment:
Can we create non-FAST keywords? (I think we can.) Does that need to be tested?

Reply:
Like, free-text ones? If so, yes we can. And sure, I'll change the second keyword here to a free-text one.